routes/customer.js

```diff
@@ -0,0 +1,49 @@
+var Ingredient = require('../model/ingredients');
+var Order = require('../model/orders');
+var Burger = require('../model/burger');
+
+module.exports = function (req, res) {
```

Comment:
This is kind of a weird style. I would make a separate property for each function, not use the main object for one function and a property for another.